docs/specs/moniker.yml

```diff
@@ -445,8 +445,6 @@ outputs:
       "conceptual": "<p>Moniker: netcore-2.0</p>",
       "monikers": ["netcore-2.0"]
     }
-  xrefmap.json: | 
-    {"references":[{"uid":"a","href":"https://docs.com/docs/a","name":"netcore-2.0"}]}
```

Comment:
@928PJY is this for testing monikers?

Reply:
we do not output moniker information in xref map for now